custom/import_xml.php

```diff
@@ -190,24 +193,29 @@ function setInsurance($pid, $ainsurance, $asubscriber, $seq) {
 ?>
 <html>
 <head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<title><?php xl('Import Patient Demographics','e'); ?></title>
+<?php Header::setupHeader(); ?>
+<title><?php xlt('Import Patient Demographics'); ?></title>
```

Comment:
Agree with removing the e parameter in all the xl() functions since this is deprecated and not secure (does not protect against cross scripting).
When you do this, though, then need to echo it, so should be:
``````
<title><?php echo xlt('Import Patient Demographics'); ?></title>